src/mines.js

```diff
@@ -0,0 +1,397 @@
+  checkpoint = true;
+  document
+    .querySelector("#newGame")
+    .classList.add("game-controll__btn--hidden");
+  turn();
+}
+
+function userWins() {
+  let amountOfCellsOpened = 0;
+  document.querySelectorAll(".cell").forEach((cell) => {
+    if (cell.classList.contains("cell-opened")) {
+      amountOfCellsOpened += 1;
+    }
+  });
+  if (amountOfCellsOpened === rows * columns - amountOfMines) {
+    return true;
+  }
+  return false;
+}
+
+function revealNeighbourCells(cellRow, cellCol) {
+  console.log(cellRow, cellCol);
+  if (
+    document.querySelector(`[data-order = '${cellRow}_${cellCol}']`) !== null
+  ) {
+    document
+      .querySelector(`[data-order = '${cellRow}_${cellCol}']`)
+      .classList.add("visited");
+  }
+  if (cellRow === rows || cellCol === columns || cellRow < 0 || cellCol < 0) {
+    return;
+  }
+  if (
+    document
+      .querySelector(`[data-order = '${cellRow}_${cellCol}']`)
+      .getAttribute("data-amount") > 0
+  ) {
+    revealCell(cellRow, cellCol);
+    return;
+  }
+  if (
+    !document
+      .querySelector(`[data-order = '${cellRow}_${cellCol}']`)
+      .classList.contains("cell-opened")
+  ) {
+    revealCell(cellRow, cellCol);
+    revealNeighbourCells(cellRow - 1, cellCol);
+    revealNeighbourCells(cellRow, cellCol - 1);
+    revealNeighbourCells(cellRow, cellCol + 1);
+    revealNeighbourCells(cellRow + 1, cellCol);
+  }
+}
+
+function gameOver() {
+  document
+    .querySelectorAll(".bomb-pic")
+    .forEach((bomb) => bomb.classList.add("shown"));
+  document
+    .querySelector("#board")
+    .insertAdjacentHTML(
+      "beforeend",
+      "<div class = 'game-status'><span class = 'game-status_text'>Press New Game button to start a new game</span></div>"
+    );
+  bombed = true;
+  document
+    .querySelector("#newGame")
+    .classList.remove("game-controll__btn--hidden");
+}
+
+function turn() {
+  seconds = 0;
+  minutes = 0;
+  bombed = false;
+  clearInterval(timerVar);
+  timerVar = setInterval(() => {
+    seconds += 1;
+    if (seconds >= 60) {
+      minutes += 1;
+      seconds = 0;
+    }
+    if (bombed) {
+      clearInterval(timerVar);
+    }
+    if (win) {
+      if (minutes > 10 && seconds > 10) {
+        document
+          .querySelector("#winTimes")
+          .insertAdjacentHTML(
+            "beforeend",
+            `<span class = "attemptsBox">00:${minutes}:${seconds}</span>`
+          );
+      } else if (minutes < 10 && seconds < 10) {
+        document
+          .querySelector("#winTimes")
+          .insertAdjacentHTML(
+            "beforeend",
+            `<span class = "attemptsBox">00:0${minutes}:0${seconds}</span>`
+          );
+      } else if (minutes < 10 && seconds > 10) {
+        document
+          .querySelector("#winTimes")
+          .insertAdjacentHTML(
+            "beforeend",
+            `<span class = "attemptsBox">00:0${minutes}:${seconds}</span>`
+          );
+      } else if (minutes > 10 && seconds < 10) {
```

Comment:
I see you're formatting numbers here. Try [String.padStart](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart) instead.